docs/blog/2023-12-12-podman-image-builds/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: '[Community] Podman-in-Podman image builds'
+description: Build images in Podman with buildah
+slug: podman-image-builds
+authors:
+  - name: handlebargh
+    url: https://github.com/handlebargh
+    image_url: https://github.com/handlebargh.png
+hide_table_of_contents: true
+tags: [community, image, podman]
+---
+
+<!-- cspell:ignore buildah Containerfile roundcube -->
+
+I run Woodpecker CI with podman backend instead of docker and just figured out how to build images with buildah. Since I couldn't find this anywhere documented I thought I might as well just share it here.
```

Comment:
```suggestion
I run Woodpecker CI with podman backend instead of docker and just figured out how to build images with buildah. Since I couldn't find this anywhere documented, I thought I might as well just share it here.
```